stacks/bindata-non-static.go

```diff
@@ -0,0 +1,11 @@
+// +build non_static
```

Comment:
If we are adding build tags that break golangci-lint then we need to also update the golangci-lint configuration

Reply:
@daxmc99 Adding `!static` tag so that by default If you run from IDE it will read localfile instead of bindata